utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -514,7 +783,16 @@ inline void write_image_batch_opencv(string outputFolder, Rpp8u *output, RpptDes
             cvtColor(matOutputImageRgb, matOutputImage, COLOR_RGB2BGR);
         }
 
-        imwrite(outputImagePath, matOutputImage);
+        fs::path pathObj(outputImagePath);
+        if (fs::exists(pathObj))
+        {
+            std::string outPath = outputImagePath.substr(0, outputImagePath.find_last_of('.')) + "_" + to_string(cnt) + outputImagePath.substr(outputImagePath.find_last_of('.'));
+            imwrite(outPath, matOutputImage);
+            cnt++;
+        }
+        else
```

Comment:
why do we need a if else here?
How else part is handling the failure cases?

Reply:
It will be useful when the user gives a batch size greater than the number of images in a folder. In this case, the last image name will be replicated to fill the batch size. If we use the same name while dumping with OpenCV, it will overwrite the outputs that are already present. To avoid this overwriting, I have used an if-else condition here.